tests/fixtures/node-app/package-lock.json

```diff
@@ -0,0 +1,115 @@
+{
```

Comment:
Do we need to have this file in the examples? If not, let's simplify it and drop it.

Reply:
It is required for the `npm ci` command, but I just updated the command to be `npm install` and removed this file to simplify it